.circleci/config.yml

```diff
@@ -246,9 +273,116 @@ jobs:
+            python .circleci/setup/generate-host-envs.py
+
+      - run:
+          name: OLD -- Build container images
+          working_directory: ~/previous_version/.circleci
+          command: |
+            #docker build --file ~/project/previous_version/deploy/Dockerfile-django --tag scionlab_django ~/project/ # We are only extending the production image
+            sed -i '/.circleci/d' ~/previous_version/.dockerignore  # The .dockerignore is for production, we'll need this
+            docker-compose build
+
+
+      - run:
+          name: OLD -- Start coordinator
+          working_directory: ~/previous_version/.circleci
+          <<: *start_coordinator
+
+      - run:
+          name: OLD -- Check coordinator ready
+          working_directory: ~/previous_version/.circleci
+          <<: *check_coordinator_ready
+
+      - run:
+          name: OLD -- Start ASes
+          working_directory: ~/previous_version/.circleci
+          command: |
+            set -x
+            docker-compose up -d as1301 as1303 as1305
+
+            for c in as1301 as1303 as1305; do
+              docker-compose exec $c /bin/bash -c \
+                'apt-get install --assume-yes scionlab; \
+                 scionlab-config --host-id ${SCIONLAB_HOST_ID} --host-secret ${SCIONLAB_HOST_SECRET} --url http://coord:8000'
+            done
+
+            docker-compose logs
+
+      - run:
+          name: OLD -- Check AS services status
+          working_directory: ~/previous_version/.circleci
+          <<: *check_services_status
+
+      # Starting upgrade integration tests
+
+      - checkout
+
+      - run:
+          name: NEW -- Rebuild and start coordinator from current branch
+          working_directory: ~/project/.circleci
+          command: |
+            sed -i '/.circleci/d' ~/project/.dockerignore  # The .dockerignore is for production, we'll need this
+            docker build --file ~/project/deploy/Dockerfile-django --tag scionlab_django ~/project/
+            sudo chown circleci:circleci -R ~/previous_version/run/
+            cp -r ~/previous_version/run ~/project/
+            docker-compose up -d --force-recreate --build coord huey
+
+      - run:
+          name: NEW -- Check coordinator ready
+          working_directory: ~/project/.circleci
+          <<: *check_coordinator_ready
+
+      - run:
+          name: OLD -- Run client cronjob
+          working_directory: ~/previous_version/.circleci
+          command: |
+            set -x
+            # Simulate command that is executed on hosts with upgrade cronjob (VMs), upgrade package
+            for c in as1301 as1303 as1305; do
+              docker-compose exec -T $c /bin/bash -c \
+                'echo "deb [trusted=yes] https://packages-test.netsec.inf.ethz.ch/debian all main" | sudo tee /etc/apt/sources.list.d/scionlab.list; \
+                 apt-get update; \
+                 rm /var/lib/scion/*; sudo mv /.dockerenv /.dockerenv.bk; \
```

Comment:
Could you please add a short comment to describe why removing the dockerenv file? 
And a #TODO or something to remove the `rm /var/lib/scion/*` once this is done as part of the package update.

Reply:
Done